frontend/src/App.vue

```diff
@@ -73,16 +73,9 @@
     <router-view></router-view>
     <ErrorPanel :message="errorMessage" @hideErrorPanel="errorMessage=''" />
     <footer id="footer" class="footer has-background-primary-lighter is-size-6 py-4">
-      <div class="columns is-gapless">
-        <div class="column is-7-desktop is-5-tablet has-text-centered-mobile">
-          <p>2021 ©
-            <span class="is-hidden-tablet-only">
-              &nbsp;Department of Biology and Biological Engineering |
-            </span>
-            &nbsp;Chalmers University of Technology</p>
-        </div>
-        <div class="column">
-          <div class="content has-text-right has-text-centered-mobile is-size-2-mobile">
+      <div class="columns columns is-gapless is-multiline">
```

Comment:
This is a bit unexpected, not only because of the duplicated `columns`, but also of the expected Bulma layout where a `columns` parent has one or more `column` children.

Reply:
Woops, nicely spotted. This is a legit fix or were you going for something else as well?
```suggestion
      <div class="columns is-gapless is-multiline">
```

Reply:
I think that would not be enough - I don't see the html children nodes with the class `column` ([more info on Bulma](https://bulma.io/documentation/columns/basics/)).

Reply:
Sorry, I don't know where I was looking - the children `column` exist.